core/consensus/pow/src/lib.rs

```diff
@@ -63,6 +63,8 @@ fn aux_key(hash: &H256) -> Vec<u8> {
 /// Auxiliary storage data for PoW.
 #[derive(Encode, Decode, Clone, Debug, Default)]
 pub struct PowAux {
+	/// Difficulty.
+	pub difficulty: Difficulty,
```

Comment:
What difficulty is this? The current difficulty?

Reply:
Yeah, it's the current block's difficulty.

Reply:
Can we reflect this in the comment?

Reply:
Fixed!